ql/src/test/results/clientnegative/delete_non_acid_table.q.out

```diff
@@ -34,4 +34,4 @@ POSTHOOK: Input: default@not_an_acid_table2
 -1070883071	0ruyd6Y50JpdGRf6HqD
 -1070551679	iUR3Q
 -1069736047	k17Am8uPHWk02cEf1jet
-FAILED: SemanticException [Error 10297]: Attempt to do update or delete on table default.not_an_acid_table2 that is not transactional
+FAILED: SemanticException [Error 10297]: Attempt to do update or delete on table not_an_acid_table2 that is not transactional
```

Comment:
we are loosing the qualified database name from the exception; I think this should be made consistent; to show it everywhere - I'm not sure about if we should be also showing the catalog name or not... but if we do; we should "try" to be consistent about that as well